lms/envs/common.py

```diff
@@ -2950,3 +2950,7 @@
 # The cache is cleared when Redirect models are saved/deleted
 REDIRECT_CACHE_TIMEOUT = None  # The length of time we cache Redirect model data
 REDIRECT_CACHE_KEY_PREFIX = 'redirects'
+
+############## Settings for LMS Context Sensitive Help ##############
+
+DOC_LINK_BASE_URL = False
```

Comment:
It shouldn't matter functionally but settings this to `None` instead might clear up potential confusion about whether this setting is a boolean or a string.